Content.Shared/SimpleStation14/CCVar/CCVars.cs

```diff
@@ -142,4 +147,13 @@ public sealed class SimpleStationCCVars
     public static readonly CVarDef<float> SiliconNpcUpdateTime =
         CVarDef.Create("silicon.npcupdatetime", 1.5f, CVar.SERVERONLY);
     #endregion Silicons
+
+    #region SSDIndicator
+    /// <summary>
+    ///     Whether or not a sprite overlay shows above SSD players
+    ///     Will not update if changed mid-round, leaving the overlays in whatever state they were
```

Comment:
Fix this. You can react to a cvar change, just set them all to false and then do something with your iterator.